matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/content/UploadContentWorker.kt

```diff
@@ -154,8 +158,8 @@ internal class UploadContentWorker(val context: Context, params: WorkerParameter
                                 compressedFile.inputStream().use {
                                     val options = BitmapFactory.Options().apply { inJustDecodeBounds = true }
                                     val bitmap = BitmapFactory.decodeStream(it, null, options)
-                                    val fileSize = bitmap?.byteCount ?: 0
-                                    newImageAttributes = NewImageAttributes(
+                                    val fileSize = bitmap?.byteCount?.toLong() ?: compressedFile.length()
```

Comment:
Reading it again, I'm not sure it will be bug free, since the inputStream is still open here.

Reply:
it's ok, my bad